common/cmake/devices.cmake

```diff
@@ -1305,6 +1305,8 @@ function(ADD_FPGA_TARGET)
   set(OUT_SYNTH_V ${OUT_LOCAL}/${TOP}_synth.v)
   set(OUT_SYNTH_V_REL ${OUT_LOCAL_REL}/${TOP}_synth.v)
   set(OUT_FASM_EXTRA ${OUT_LOCAL}/${TOP}_fasm_extra.fasm)
+  set(OUT_SDC ${OUT_LOCAL}/${TOP}.sdc)
```

Comment:
Should be `${TOP}_synth.sdc` to be clear this is a post-synthesis SDC file.

Also worth making a comment in the CMake about what this file is

Reply:
Done